src/main/java/com/example/gradecalculator/config/PasswordConfig.java

```diff
@@ -2,14 +2,15 @@
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.factory.PasswordEncoderFactories;
 import org.springframework.security.crypto.password.PasswordEncoder;
 
 @Configuration
 public class PasswordConfig {
 
     @Bean
     public PasswordEncoder passwordEncoder() {
-        return new BCryptPasswordEncoder();
+        return PasswordEncoderFactories.createDelegatingPasswordEncoder();
     }
-}
\ No newline at end of file
+
+    }
```

Comment:
Why is there a new line (more spacing) but also worse bracket placement if this did not gave to change at all?